### PR TITLE
Added nuxt integration page.

### PR DESCRIPTION
**Pull request by m0ksem**
image

### CODE REVIEW

**Review by asvae — APPROVED** [review]
Looks good! 👍

File: packages/docs/src/components/sidebar/navigationRoutes.ts [hunk 1]
```diff
@@ -48,6 +48,10 @@ export const navigationRoutes: NavigationRoute[] = [
         name: 'tree-shaking',
         displayName: 'menu.treeShaking',
       },
+      {
+        name: 'nuxt',
+        displayName: 'menu.nuxt',
+      },
```
Comment (RVitaly1978) [inline packages/docs/src/components/sidebar/navigationRoutes.ts:54]:
```suggestion
      {
        name: 'nuxt',
        displayName: 'menu.nuxt',
        meta: {
          badge: 'new',
        },
      },
```

File: packages/docs/src/locales/ru/ru.json [hunk 2]
```diff
@@ -3723,5 +3724,14 @@
     "attention": {
       "cssLoader": "Поскольку код всех компонентов разделен, вам может потребоваться загрузчик `CSS` для загрузки стилей для конкретных компонентов.\nНо, вероятно, он у вас уже есть."
     }
+  },
+  "nuxt": {
+    "title": "Nuxt интеграция",
+    "description": "Vuestic UI компоненты полностью совместимы с Nuxt3.",
+    "installation": {
+      "title": "Установка",
+      "plugin": "Сделайте плагин файл `plugins/vuestic-ui.ts` в nuxt проекте. Он должен ключать в себя следующий код:",
```
Comment (RVitaly1978) [inline packages/docs/src/locales/ru/ru.json:3733]:
```suggestion
      "plugin": "Создайте плагин файл `plugins/vuestic-ui.ts` в nuxt проекте. Он должен ключать в себя следующий код:",
```

File: packages/docs/src/pages/getting-started/nuxt.vue [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <docs-content :config="configComputed" />
+</template>
+
+<script lang="ts">
+import { Vue, Options } from 'vue-class-component'
+import DocsContent from '../../components/DocsContent.vue'
+import nuxtConfig from '../../page-configs/getting-started/nuxt/page-config'
```
Comment (RVitaly1978) [inline packages/docs/src/pages/getting-started/nuxt.vue:8]:
```suggestion
import config from '../../page-configs/getting-started/nuxt/page-config'
```

File: packages/docs/src/pages/getting-started/nuxt.vue [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <docs-content :config="configComputed" />
```
Comment (RVitaly1978) [inline packages/docs/src/pages/getting-started/nuxt.vue:2]:
```suggestion
  <docs-content :config="config" />
```

File: packages/docs/src/pages/getting-started/nuxt.vue [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+</template>
+
+<script lang="ts">
+import { Vue, Options } from 'vue-class-component'
```
Comment (RVitaly1978) [inline packages/docs/src/pages/getting-started/nuxt.vue:6]:
```suggestion
import { defineComponent } from 'vue'
```

**Review by RVitaly1978 — APPROVED** [review]

File: packages/docs/src/pages/getting-started/nuxt.vue (hunk 5 above)
Comment (m0ksem) [reply packages/docs/src/pages/getting-started/nuxt.vue:6]: It is auto generated code. Let's let it be, so we can refactor it globally later. I used `yarn generate:docspage`

**Review by RVitaly1978 — APPROVED** [review]

File: packages/docs/src/locales/ru/ru.json [hunk 6]
```diff
@@ -3723,5 +3724,14 @@
     "attention": {
       "cssLoader": "Поскольку код всех компонентов разделен, вам может потребоваться загрузчик `CSS` для загрузки стилей для конкретных компонентов.\nНо, вероятно, он у вас уже есть."
     }
+  },
+  "nuxt": {
+    "title": "Nuxt интеграция",
+    "description": "Vuestic UI компоненты полностью совместимы с Nuxt3.",
+    "installation": {
+      "title": "Установка",
+      "plugin": "Создайте плагин файл `plugins/vuestic-ui.ts` в nuxt проекте. Он должен ключать в себя следующий код:",
```
Comment (Derranion) [inline packages/docs/src/locales/ru/ru.json:3733]:
```suggestion
      "plugin": "Создайте плагин файл `plugins/vuestic-ui.ts` в nuxt проекте. Он должен включать в себя следующий код:",
```

**Review by Derranion — APPROVED** [review]

**Pull request merged**